src/EmailCampaigns/NewPublishPost/NewPublishPost.php

```diff
@@ -113,11 +113,7 @@ public function new_publish_post($new_status, $old_status, $post)
 
                 if (ER::is_campaign_active($email_campaign_id) === false) continue;
     
-                //post translations
-                $post = apply_filters('mailoptin_email_campaign_translated_post',
-                    $post,
-                    $email_campaign_id
-                );
+                if(ER::has_current_language($post, $email_campaign_id) === false) continue;
```

Comment:
@ShoboySnr oh oh! you have started again ☹️

Instead of this, apply a filter here that you can use to skip the process something like

if(apply_filters('mo_new_publish_post_shoud_skip', false)) continue;

Reply:
Ohhh.. okay, will do that